django_fido/js/fido2.js

```diff
@@ -139,14 +139,20 @@ async function sendFido2Request(url, is_registration, credenetials_name, form_da
     }
 }
 
-async function sendFido2RegistrationRequest(url, form_data) {
-    await sendFido2Request(url, true, 'excludeCredentials', form_data)
-}
-
 async function sendFido2AuthenticationRequest(url, form_data) {
     await sendFido2Request(url, false, 'allowCredentials', form_data)
 }
 
+const sendFido2RegistrationRequest = async(e) => {
+    e.preventDefault()
+    const form = document.getElementById(DJANGO_FIDO_FORM_ID)
+    let form_data = ''
+    const user_field = form.querySelector('[name=user]')
+    if (user_field)
+        form_data = `user=${user_field.value}`
```

Comment:
Is the condition needed? I guess we can get rid of it with the use of `optional chaining`, something like:
`const form_data = form.querySelector('[name=user]')?.value`

Although that would return `undefined` instead of the empty string in case the condition is falsy. If it can't handle undefined value, it could be handled by `nullish coalescing`:
`const form_data = form.querySelector('[name=user]')?.value ?? ''`

Reply:
Yes it is necessary this javascript handles multiple forms which may not include the user field. It can also be used with "custom" form not provided by this library.

your suggestion is missing the key `user`, would this suffice?
`
const form_data = form.querySelector('[name=user]') ? ``user=${form.querySelector('[name=user]').value}`` : ''
`

Reply:
Yea, ok, this suggestion is fine.